integrations/ad_integration/ad_sync.py

```diff
@@ -75,47 +81,30 @@ def __init__(self):
             if not found:
                 raise Exception('Error in visibility class configuration')
 
+        used_mo_fields = []
```

Comment:
Alt i alt er jeg fuld af beundring for hele dette sæt af rettelser.Nu har jeg endnu ikke set om du rent faktisk har prøvet det, men en glimrende test kunne være at køre to parallelle ting en gang imellem: En, der henter fra Clora og en, der henter fra Mo - hvis man kan køre hele os2mo igennem og få samme resultat er man vel sådan set sikret at man kan bytte den ene ud med den anden. Et sådant testprogram kunne man så køre hos alle kunder hver weekend for at finde de eventualiteter, der måtte snige sig uden om.

Reply:
Det vil selvfølgelig være virkelig godt, men et ret stort projekt, hele eksporten skulle så også implementeres via MOs endpoints.